libs/oniguruma/Makefile

```diff
@@ -0,0 +1,54 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=oniguruma
+PKG_VERSION:=6.9.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=onig-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/kkos/oniguruma/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=3b568a9050839e7828b2f2d5bc9cd3650979b6b54a080f54c515320dddda06b0
+
+PKG_MAINTAINER:=Eneas U de Queiroz <cote2004-github@yahoo.com.br>
+PKG_LICENSE:=BSD-2-Clause
+PKG_LICENSE_FILES:=COPYING
+PKG_CPE_ID:=cpe:/a:oniguruma_project:oniguruma
+
+PKG_INSTALL:=1
+PKG_FIXUP:=autoreconf
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/oniguruma
+    SECTION:=libs
+    CATEGORY:=Libraries
+    TITLE:=Regular expression library for different character encodings
+    URL:=https://github.com/kkos/oniguruma
+    DEPENDS:=
```

Comment:
Could be removed.

Reply:
OK.  I'll remove it after you finish your review.